$NetBSD: patch-av,v 1.1 2008/06/03 06:59:13 dsainty Exp $

'program_invocation_short_name' may be conditionally defined even if there is
a declaration available.  Make sure it's always available by making the
extern unconditional.

--- src/mesa/drivers/dri/common/xmlconfig.c.orig	2007-06-22 10:10:54.000000000 +1200
+++ src/mesa/drivers/dri/common/xmlconfig.c	2008-04-27 21:31:28.660265951 +1200
@@ -49,10 +49,8 @@
 #undef GET_PROGRAM_NAME
 
 #if (defined(__GNU_LIBRARY__) || defined(__GLIBC__)) && !defined(__UCLIBC__)
-#    if !defined(__GLIBC__) || (__GLIBC__ < 2)
 /* These aren't declared in any libc5 header */
 extern char *program_invocation_name, *program_invocation_short_name;
-#    endif
 #    define GET_PROGRAM_NAME() program_invocation_short_name
 #elif defined(__FreeBSD__) && (__FreeBSD__ >= 2)
 #    include <osreldate.h>
